compiler/prelude/jsmapping.go

```diff
@@ -245,6 +245,9 @@ var $internalize = function(v, t, recv) {
     if (v === null) {
       return $ifaceNil;
     }
+    if (v === undefined) {
+      return new $jsObjectPtr(undefined);
+    }
```

Comment:
I think the reason `a === b` doesn't hold below when comparing two instances of `undefined` may be because I'm calling `new` every time here. Maybe it should be done outside, and this can return a single/the same instance of `new $jsObjectPtr(undefined)"` every time.